website/docs/foundations/typography/partials/code/how-to-use.md

```diff
@@ -1,64 +1,58 @@
-The suggested way to apply the typographic definitions to a UI element is using the **predefined CSS helper classes** provided.
+## How to use these styles
 
-#### Design tokens
+While we do offer design tokens for typography, we recommend using the predefined CSS helper classes.
 
-There are numerous [typographic design tokens](./tokens) in our system.
+### CSS helper classes
 
-Since they are "atomic" definitions that associate a particular typographic property (e.g. `font-size`, or `line-height`, or `letter-spacing`) to a single value, they are not exactly "typographic" styles.
+In a single declaration, these classes contain everything to apply a standard style to an element: `font-family`, `font-size`, `line-height`, and a reset for `margin` and `padding` to match the intended design language.
 
-For this reason we **strongly advise against** using them directly in your CSS, and prefer the CSS helpers provided by the system, to avoid "mix & match" of typographic styles.
+```markup
+<p class="hds-typography-display-300">The fox jumped over the lazy dog.</p>
+```
 
-#### CSS helper classes
+#### Font styles
 
-There are different CSS helper classes that can be used for different purposes.
+<Doc::FontHelpersList @items={{this.cssHelpers.styles}} />
 
-If you want to change **only** the _font-family_ you can use one the "font-family" helpers:
+#### Font family
+
+While we don’t recommend using custom styles often, `font-family` helpers are available to change the font family of an element.
 
 ```markup
-<p class="hds-font-family-sans-text">Lorem ipsum dolor.</p>
+<p class="hds-font-family-sans-text">The fox jumped over the lazy dog.</p>
```

Comment:
```suggestion
<p class="hds-font-family-sans-text">The quick brown fox jumps over the lazy dog.</p>
```

Reply:
Up to you if you want to update, but this is the version that contains every letter of the alphabet.